winter/schema/query_parameters_inspector.py

```diff
@@ -1,23 +1,18 @@
-import inspect
 from typing import List
 
-import docstring_parser
 from drf_yasg import openapi
 
+from .docstring_parser import DocstringParser
 from .generation import get_argument_type_info
 from .method_arguments_inspector import MethodArgumentsInspector
-from .utils import update_doc_with_invalid_hype_hint
-from .. import type_utils
 from ..core import ComponentMethod
 from ..query_parameter import get_query_param_mapping
 
 
 class QueryParametersInspector(MethodArgumentsInspector):
 
     def inspect_parameters(self, method: ComponentMethod) -> List[openapi.Parameter]:
-        docstring = docstring_parser.parse(method.func.__doc__)
-        params_docs = {param_doc.arg_name: param_doc for param_doc in docstring.params}
-
+        docstring_parser = DocstringParser(method.func.__doc__)
```

Comment:
Please consider the following alternative: `docstring = method.docstring`. It will allow to parse docstring for each method only once.